src/js/main.js

```diff
@@ -233,12 +256,19 @@ if (document.body.dataset.page === "run") {
     window.location.href = "/";
   }
 
-  settingsStore.getAndSubscribe(({ livebookUrl }) => {
-    const livebookImportUrl = getLivebookImportUrl(livebookUrl, notebookUrl);
-    for (const runNotebookLinkEl of runNotebookLinkEls) {
-      runNotebookLinkEl.setAttribute("href", livebookImportUrl);
+  settingsStore.getAndSubscribe(({ livebookUrl, useLivebookDesktop }) => {
+    if (useLivebookDesktop) {
+      for (const runNotebookLinkEl of runNotebookLinkEls) {
+        runNotebookLinkEl.setAttribute("href", notebookUrl.replace(/^https?:/i, "livebook:"));
+      }
+      livebookUrlEl.textContent = "Livebook for desktop";
+    } else {
+      const livebookImportUrl = getLivebookImportUrl(livebookUrl, notebookUrl);
+      for (const runNotebookLinkEl of runNotebookLinkEls) {
+        runNotebookLinkEl.setAttribute("href", livebookImportUrl);
+      }
+      livebookUrlEl.textContent = livebookUrl;
```

Comment:
We could extract the common part to `updateRunNotebookLinks(url, caption)` :)

Reply:
done